packages/terra-worklist-data-grid/src/subcomponents/Row.module.scss

```diff
@@ -3,12 +3,18 @@
 @import '../clinical-lowlight-theme/Row.module';
 
 :local {
+  tr.selectable:hover {
```

Comment:
You could really create a parent .worklsit-data-grid-row selector to group all your styles and then use your parent operator (&) to handle these conditions.